JAICore/jaicore-ml/src/main/java/ai/libs/jaicore/ml/scikitwrapper/ScikitLearnRULWrapper.java

```diff
@@ -0,0 +1,16 @@
+package ai.libs.jaicore.ml.scikitwrapper;
+
+import java.io.IOException;
+
+import org.api4.java.ai.ml.core.evaluation.IPrediction;
+import org.api4.java.ai.ml.core.evaluation.IPredictionBatch;
+
+import ai.libs.jaicore.ml.core.EScikitLearnProblemType;
+
+public class ScikitLearnRULWrapper<P extends IPrediction, B extends IPredictionBatch> extends ScikitLearnRegressionWrapper<P, B> {
+
+	public ScikitLearnRULWrapper(final String pipeline, final String imports) throws IOException {
+		super(EScikitLearnProblemType.RUL, pipeline, imports);
+	}
+
```

Comment:
Handling of output?

Reply:
Is equivalent to ScikitLearnRegressionWrapper, that is why RUL extends Regression